src/Bijectors.jl

```diff
@@ -190,6 +192,44 @@ function link(
     return y
 end
 
+function link_jacobian(
+    d::SimplexDistribution, 
+    x::AbstractVector{T}, 
+    ::Type{Val{proj}} = Val{true}
+) where {T<:Real, proj}
+    y, K = similar(x), length(x)
+    dydxt = similar(x, length(x), length(x))
+    @inbounds dydxt .= 0
+    ϵ = _eps(T)
+    sum_tmp = zero(T)
+
+    @inbounds z = x[1] * (one(T) - 2ϵ) + ϵ # z ∈ [ϵ, 1-ϵ]
+    @inbounds y[1] = StatsFuns.logit(z) + log(T(K - 1))
+    @inbounds dydxt[1,1] = (1/z + 1/(1-z)) * (one(T) - 2ϵ)
+    @inbounds @simd for k in 2:(K - 1)
+        sum_tmp += x[k - 1]
+        # z ∈ [ϵ, 1-ϵ]
+        # x[k] = 0 && sum_tmp = 1 -> z ≈ 1
+        z = (x[k] + ϵ)*(one(T) - 2ϵ)/((one(T) + ϵ) - sum_tmp)
+        y[k] = StatsFuns.logit(z) + log(T(K - k))
+	    dydxt[k,k] = (1/z + 1/(1-z)) * (one(T) - 2ϵ)/((one(T) + ϵ) - sum_tmp)
+	    for i in 1:k-1
```

Comment:
I added some comments for my own understanding. Leaving them here in case this is something we'd like to put in the code:
```
	for i in 1:k-1
            # ∂ᵢlogit(zₖ) = (1 / zₖ + 1 / (1 - zₖ)) (∂ᵢ zₖ)
            # ∂ᵢzₖ = ∂ᵢ((xₖ + ϵ) * (1 - 2 ϵ) / (1 + ϵ - sum_tmp))
            #      = (xₖ + ϵ) * (1 - 2 ϵ) (∂ᵢ (1 + ϵ - sum_tmp)⁻¹)
            #      = (xₖ + ϵ) * (1 - 2 ϵ) (-1) (1 + ϵ - sum_tmp)⁻² ∂ᵢ (1 + ϵ - sum_tmp)
            #      = (xₖ + ϵ) * (1 - 2 ϵ) (-1) (1 + ϵ - sum_tmp)⁻² (-1) (∂ᵢ sum_tmp)
            #      = (xₖ + ϵ) * (1 - 2 ϵ) (1 + ϵ - sum_tmp)⁻² ∂ᵢ (∑ⱼᵏ⁻¹ xⱼ)
            #      = (xₖ + ϵ) * (1 - 2 ϵ) (1 + ϵ - sum_tmp)⁻² (∑ⱼᵏ⁻¹ δᵢʲ)
            #      = (xₖ + ϵ) * (1 - 2 ϵ) (1 + ϵ - sum_tmp)⁻² ⋅ 1
            #      = (xₖ + ϵ) * (1 - 2 ϵ) (1 + ϵ - sum_tmp)⁻²                            ■
	    dydxt[i,k] = (1/z + 1/(1-z)) * (x[k] + ϵ)*(one(T) - 2ϵ)/((one(T) + ϵ) - sum_tmp)^2
```

Reply:
What's the character after `x` and `z`? I cannot see then on my computer.